chain/src/chain.rs

```diff
@@ -827,12 +871,17 @@ impl BlockChain {
                 None => (uncles, None),
                 Some(tips) => {
                     let mut blues = self.dag.ghostdata(tips)?.mergeset_blues.to_vec();
+                    if blues.is_empty() {
```

Comment:
Return error instead of bailling out?

Reply:
Is it different? bail is used to return the error earlier.